website/static/js/logTextParser.js

```diff
@@ -223,7 +224,7 @@ var LogPieces = {
         view: function (ctrl, logObject) {
             var destination = logObject.attributes.params.destination;
             if(paramIsReturned(destination, logObject)){
-                return m('a', {href: destination.url}, destination.materialized);
+                return m('span', [m('a', {href: destination.url}), destination.materialized, ' in ', source.addon]);
```

Comment:
Also, not sure if this is related, but project logs seem to only link the "destination" file:
![screen shot 2016-02-22 at 10 51 09 am](https://cloud.githubusercontent.com/assets/6414394/13223596/e9d139ca-d952-11e5-9c2a-3415e882ee7c.png)

But recentActivity logs link the "source" file:
![screen shot 2016-02-22 at 10 51 24 am](https://cloud.githubusercontent.com/assets/6414394/13223593/e78995fe-d952-11e5-9d97-20d367ce90eb.png)

Also, these project logs include the name of the project. I think it would be useful if the recent activity logs included that info as well. Especially because it's aggregating logs from multiple projects.